compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/converters/URIDryConverter.java

```diff
@@ -98,73 +100,77 @@ public URIDryConverter(URI base, Map<PackageID, Manifest> dependencyManifests, b
             // ignore errors
         }
     }
-    
+
     public Stream<CompilerInput> finalize(URI remoteURI, PackageID moduleID) {
         try {
             // only continue if a fixed module version is not set. a module version may be set through Ballerina.toml or
             // Ballerina.lock already.
             Matcher matcher = semVerPatchPattern.matcher(moduleID.version.value);
             if ("".equals(moduleID.version.value) || "*".equals(moduleID.version.value) || matcher.matches()) {
-                for (String supportedPlatform : supportedPlatforms) {
-                    HttpURLConnection conn;
-                    // set proxy if exists.
-                    if (null == this.proxy) {
-                        conn = (HttpURLConnection) remoteURI.toURL().openConnection();
-                    } else {
-                        conn = (HttpURLConnection) remoteURI.toURL().openConnection(this.proxy);
-                    }
-                    conn.setInstanceFollowRedirects(false);
-                    conn.setRequestMethod("GET");
-                    // set implementation version
-                    conn.setRequestProperty("Ballerina-Platform", supportedPlatform);
-                    conn.setRequestProperty("Ballerina-Language-Specification-Version", IMPLEMENTATION_VERSION);
```

Comment:
What is the reason for removing the above two properties?

Reply:
These headers are not needed to access this endpoint.